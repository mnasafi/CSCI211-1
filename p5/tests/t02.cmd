2 0 tests/t02.in results/t02.myout
