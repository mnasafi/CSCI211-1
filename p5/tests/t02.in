Maggie shopper 2 2
Marge shopper 4 2
Lisa shopper 3 2
Bart robber 5 2
Homer shopper 1 2
