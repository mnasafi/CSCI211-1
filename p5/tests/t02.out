1: Homer entered store
2: Maggie entered store
3: Lisa entered store
4: Marge entered store
5: Bart entered store
5: Homer done shopping
5: Homer started checkout with checker 0
6: Maggie done shopping
6: Maggie started checkout with checker 1
7: Lisa done shopping
7: Homer paid $6 for 2 items to checker 0
7: Lisa started checkout with checker 0
8: Marge done shopping
8: Maggie paid $6 for 2 items to checker 1
8: Marge started checkout with checker 1
9: Bart done shopping
9: Lisa paid $6 for 2 items to checker 0
9: Bart started checkout with checker 0
10: Marge paid $6 for 2 items to checker 1
13: Bart stole $112 and 2 items from checker 0
registers[0] = $0
registers[1] = $112
time = 14
